src/test/package/nuget/NuGetTest/TestRuntimeComponent2/module.cpp

```diff
@@ -0,0 +1,56 @@
+#include "pch.h"
+#include "winrt/base.h"
+
+bool __stdcall winrt_can_unload_now() noexcept
+{
+    if (winrt::get_module_lock())
+    {
+        return false;
+    }
+
+    winrt::clear_factory_cache();
+    return true;
+}
+
+void* __stdcall winrt_get_activation_factory(std::wstring_view const& name)
+{
+    void* factory = TestRuntimeComponent2_get_activation_factory(name);
```

Comment:
@kennykerr Is this what you were thinking with -lib? I'm getting some identifier not found error which makes sense as I don't have a separate definition, but wondering how you imagined this to work.